src/scripts/main.js

```diff
@@ -1,13 +1,30 @@
 'use strict';
 
+const form = document.querySelector('form');
+
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve, reject) => {
+    if (element) {
+      element.addEventListener(eventName, () => {
+        const string = `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`;
+
+        resolve(string);
+      });
+    } else{
+      reject(new Error('Element doesn\'t exist'));
+    }
+  });
 }
 
+const messageDiv = document.createElement('div');
+messageDiv.classList.add('message');
+messageDiv.style.display = 'none';
 const printMessage = (message) => {
-  // write your code here
+  messageDiv.innerText = message;
+  messageDiv.style.display = 'block';
 };
 
+
 const loginField = document.getElementById('login');
```

Comment:
These variables are declared but never used. Consider removing them if they're not needed.